engine.py

```diff
@@ -1352,11 +1361,11 @@ def __send_state(self):
         jump_commands = []
 
         # the icon to use for the command. bundled with the engine
-        sg_icon = os.path.join(
-            self.disk_location,
-            "resources",
-            "shotgun_logo.png"
-        )
+        # sg_icon = os.path.join(
```

Comment:
did you comment it out because it wasn't working, wasn't needed?

Reply:
yes, flake8 was complaining about it. I thought it was better to keep it with the commented code below.